data/hex-base64-encoding.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * @title Hex and Base64 Encoding
+ * @difficulty beginner
+ * @tags cli
+ * @run <url>
+ *
+ * There are a few cases where it would be practical to encode
+ * and decode between different string and array buffer formats.
+ * The Deno standard library makes this easy.
+ */
+
+// The standard library provides hex and base64 encoding and decoding utilities
+import * as base64 from "https://deno.land/std@0.171.0/encoding/base64.ts";
+import * as hex from "https://deno.land/std@0.171.0/encoding/hex.ts";
+
+// We can easily encode a string or an array buffer into base64 using the base64.encode method.
+const base64_encoded = base64.encode("somestringtoencode");
```

Comment:
I noticed in general that there's some mix between `camelCase` and `snake_case`. Given the other examples in this repo, it would be best to switch to `camelCase`.